src/openvic-simulation/map/Map.cpp

```diff
@@ -577,6 +602,7 @@ bool Map::load_map_images(fs::path const& province_path, fs::path const& terrain
 bool Map::_generate_province_adjacencies() {
 	bool changed = false;
 
+
```

Comment:
Could you remove the changes to this function, as it's not actually being changed in this PR (this newline + the `province_index` variable in the loop)

Reply:
Yes. Not sure how this ended up changed.